src/WebJobs.Extensions.DurableTask/Bindings/BindingHelper.cs

```diff
@@ -70,5 +75,17 @@ public Task FlushAsync(CancellationToken cancellationToken = default(Cancellatio
                 return Task.CompletedTask;
             }
         }
+
+        public class OrchestrationClientData
+        {
+            [JsonProperty("taskHubName")]
+            public string TaskHubName { get; set; }
+
+            [JsonProperty("creationPayload")]
+            public HttpCreationPayload CreationPayload { get; set; }
+
+            [JsonProperty("managementPayload")]
+            public HttpManagementPayload ManagementPayload { get; set; }
```

Comment:
nit: I'm not a fan of the "payload" suffix because it's not really meaningful. The class name cannot be changed (that would be breaking), but what are your thoughts on changing the field names to something like `CreationUrls` and `ManagementUrls`?